### PR TITLE
Fysetc 2.0.x

### PR DESCRIPTION
**Pull request by GerogeFu**
### Description
1. Update FYSETC STM32F1 BOARD example cnofiguration
2. FYSETC STM32F1 BOARD pins update
3. Change LCD com defines for STM32
4. Fix STM32F1 st7920 bug
5. Add FYSETC STM32F1 boards upload script
6. Add FYSETC CHEETAH board BLTOUCH config example

### CODE REVIEW

File: Marlin/src/lcd/dogm/ultralcd_st7920_u8glib_rrd_AVR.cpp [hunk 1]
```diff
@@ -42,11 +42,19 @@
...
+    WRITE(ST7920_DAT_PIN, (val & 0x80)?HIGH:LOW); ST7920_DELAY_2; \
+    WRITE(ST7920_CLK_PIN, HIGH);                  ST7920_DELAY_3; \
+    val <<= 1
+#else
+  #define ST7920_SND_BIT \
+    WRITE(ST7920_CLK_PIN, LOW);        ST7920_DELAY_1; \
+    WRITE(ST7920_DAT_PIN, val & 0x80); ST7920_DELAY_2; \
+    WRITE(ST7920_CLK_PIN, HIGH);       ST7920_DELAY_3; \
+    val <<= 1
+#endif
```
Comment (tpruvot) [inline Marlin/src/lcd/dogm/ultralcd_st7920_u8glib_rrd_AVR.cpp]: maybe a single line fix with bool(val & 0x80) or (val & 0x80) != 0 ?

Comment (GerogeFu) [reply Marlin/src/lcd/dogm/ultralcd_st7920_u8glib_rrd_AVR.cpp]: Yes , i test the code , it's fine

**Comment by thinkyhead** [comment]
Oops, you force-pushed without first doing:
```
git fetch origin
git reset --hard origin/fysetc-2.0.x
```

I still have the rebased and patched branch if you want it.
https://github.com/thinkyhead/Marlin/tree/bf2_fysetc_more_14665

**Comment by GerogeFu** [comment]
Sorry , my bad operation. Do that please. Thanks. Do i need to do anything to rescue the bad operation?

**Pull request closed without merging**